### PR TITLE
fix: Displaying filter indicators

### PR DESCRIPTION
**Pull request by agatapst**
### SUMMARY
The problem was that chart indicator did not match native filter indicator and this PR fixes that.
Be aware that there is still filter component next to the charts. It is separated from the native filters, but has impact on the same charts. It also counts as "Applied filter".

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before:
indicator_1_before
image

After: 
indicator_1_after
indicator_2_after

### TEST PLAN
Verify manually. Go to `config.py` and set `"DASHBOARD_NATIVE_FILTERS": True`
Go to a dashboard and create Native Filter. 
Click an indicator presented on the charts (dark gray pill with a number). See if the information matches your filter.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

cc @junlincc @villebro 
@adam-stasiak could you please test?

### CODE REVIEW

**Comment by adam-stasiak** [comment]
@agatapst I can see situation where child filters are not displayed as set when I use them:
Zrzut ekranu 2021-01-5 o 11 55 17
Zrzut ekranu 2021-01-5 o 11 55 23
I think it should be displayed under APPLIED FILTERS section

**Comment by adam-stasiak** [comment]
After second test it is ok! no issue from above message

**Review by junlincc — APPROVED** [review]
LGTM! 
After
Screen Shot 2021-01-12 at 8 56 52 AM


Before
Screen Shot 2021-01-12 at 8 31 42 AM


NOTE⚠️: 

- This is only Product and QA approval with NO code reviews 
- This approval indicates that all UI changes meet product requirements, no noticeable regression 
- This approval gives the green light to code review, thus saving the PMC's effort from reviewing the code repeatedly before the PR is ready
- Please do NOT merge this PR without at least one code review approval from PMC member
- Please do NOT merge PRs in(Explore and Dashboard) without Product and QA approval

File: superset-frontend/src/dashboard/containers/FiltersBadge.tsx
Comment (suddjian) [inline superset-frontend/src/dashboard/containers/FiltersBadge.tsx:54]: I would suggest using `_.uniqBy()` here instead of `reduce()`.

**Review by rusackas — APPROVED** [review]
LGTM!

File: superset-frontend/src/dashboard/containers/FiltersBadge.tsx
Comment (suddjian) [reply superset-frontend/src/dashboard/containers/FiltersBadge.tsx:54]: On second thought, you'd probably have to use `uniqWith`, not `uniqBy`.

**Comment by rusackas** [comment]
Pinged @suddjian for feedback, and he left some seconds before I approved! I think it's good advice if you don't mind the quick tweak.

File: superset-frontend/src/dashboard/containers/FiltersBadge.tsx
Comment (agatapst) [reply superset-frontend/src/dashboard/containers/FiltersBadge.tsx]: Thanks, that's a nice tip!

**Pull request merged**
